apps/web-mzima-client/src/app/shared/components/company-info/company-info.component.html

```diff
@@ -9,7 +9,14 @@
     <mat-panel-title class="company-info__header">
       <img [src]="logo" [alt]="title" class="company-info__logo" *ngIf="logo" />
       <span class="company-info__title">{{ title }}</span>
-      <mat-icon svgIcon="arrow-down" class="company-info__header__arrow"></mat-icon>
+      <mat-icon
```

Comment:
Preferably, for semantic reasons, you can wrap the mat-icon element in a button element, and then apply the focus-visible style on the button. Instead of adding role button to an icon element.